### ISSUE TITLE
claimTokensFromErc677 method is missing on HomeBridgeErcToErc

### ISSUE THREAD

**Issue by patitonar** [poanetwork/poa-bridge-contracts#118]
Similarly to the Foreign contract in `native-to-erc20` mode, the Home contact in `erc20-to-erc20` mode should include method `claimTokensFromErc677`.

```solidity
function claimTokensFromErc677(address _token, address _to) external onlyProxyOwner {
        erc677token().claimTokens(_token, _to);
    }
```

**Comment by akolotov** [comment]
After introduction of support ERC677 for the foreign side https://github.com/poanetwork/poa-bridge-contracts/pull/177 it is necessary to add this functionality to the foreign contract as well.

**Comment by akolotov** [comment]
Moving the issue to the project related to the security audit of the release 2.3.2.

**Comment by akolotov** [comment]
Included in [TokenBridge contracts 2.3.3](https://github.com/poanetwork/poa-bridge-contracts/releases/tag/2.3.3)
